modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -40,10 +39,7 @@ To start using Dedicated, contact https://redpanda.com/try-redpanda?section=ente
 === Bring Your Own Cloud (BYOC)
 
 With BYOC clusters, you deploy Redpanda in your own VPC, so all data is
-contained in your own environment. This provides an additional layer of security
-and isolation. Redpanda handles provisioning, operations, and maintenance. See <<BYOC architecture>>.
-
-Available on AWS or GCP. 
+contained in your own environment. This provides an additional layer of security and isolation. When you create a BYOC cluster, you select a xref:deploy:deployment-option/cloud/byoc-tiers.adoc[throughput tier and region] on either AWS or GCP. Redpanda handles provisioning, operations, and maintenance. See also: <<BYOC architecture>>.
 
 ==== Sign up for BYOC
```

Comment:
Is this info needed for an overview page? It seems a little crammed in there.

Reply:
We need a comparison somewhere between the three different cloud managed deployment models. 

I think it's reasonable here.